normalizer_lib_test.py

```diff
@@ -3,19 +3,21 @@
 
 import unittest
 from typing import List
+from pynini.lib import rewrite
 import normalizer_lib
 import preprocess
 
+NORM = normalizer_lib.NormalizerLib("zu")
```

Comment:
This is totally a nit, but this is in CAPS case here, but lower case in "normalizer.py".  Should that be the same?  :)

Reply:
The beauty of pylint. Pylint thinks `NORM` here is a constant, so it should be in CAPS. In `normalizer.py` `norm` pytlint thinks it's a variable, so it should be snake_case. I'm all for ignoring pylint's suggestions though in this case if it adds consistency :)

Reply:
Haha, thanks Pylint!  I think consistency is a good way to go :).  I'll leave it up to you!

Reply:
I decided to lowercase this one to match `normalizer.py`.